dolphinscheduler-e2e/dolphinscheduler-e2e-case/src/test/java/org/apache/dolphinscheduler/e2e/cases/workflow/BaseWorkflowE2ETest.java

```diff
@@ -67,6 +71,14 @@ public static void setup() {
                     .goToTab(UserPage.class)
                     .update(adminUser);
         }
+        tenantPage
+                .goToNav(SecurityPage.class)
+                .goToTab(EnvironmentPage.class)
+                .createEnvironmentUntilSuccess(pythonEnvironment.getEnvironmentName(),
+                        pythonEnvironment.getEnvironmentConfig(),
+                        pythonEnvironment.getEnvironmentDesc(),
+                        pythonEnvironment.getEnvironmentWorkerGroup());
```

Comment:
Can we move this in `PythonTaskE2ETest`? Since this is not need by other tasks.

Reply:
Since this step is the front task for the entire python test, `BeforeAll` is already used.